src/cats/BreedSelector.js

```diff
@@ -0,0 +1,34 @@
+import "./BreedSelector.css";
+
+function BreedSelector({ dispatch, breeds, selectedBreedID, status }) {
+  function onSelectChange(e) {
+    dispatch({ type: "select-breed", id: e.target.value });
+  }
+
+  return (
+    <div className="BreedSelector">
+      <form>
+        <label>Breed</label>
+        <span className="selectContainer">
+          <select
+            className={status === "error" ? "isError" : ""}
+            disabled={breeds === null || status === "error"}
+            onChange={onSelectChange}
+            type="select"
+            value={selectedBreedID || undefined}
```

Comment:
The ID is a string (e.g. "all"). It is initialized to null in the parent component, but the select wants it to be a valid value or undefined. I don't like initializing to undefined, so this is the workaround to avoid the error.